### PR TITLE
Wolfram Coding Challenge Video

### PR DESCRIPTION
**Pull request by shiffman**
This replaces #1354, I wanted to work on this in a branch in this repo rather than a fork, not sure if there is a better way to do so. This pull request retains all of the commits!

### CODE REVIEW

File: content/pages/challenges/index.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
   "title": "Challenges",
   "description": "Ready to apply what you’ve learned in the Tracks? Try a Challenge! These one-off project videos build off concepts introduced in Tracks and may have prerequisites (listed on the challenge page itself).",
   "featuredText": "Featured Challenge:",
-  "featuredChallenge": "127-brownian-tree-snowflake"
+  "featuredChallenge": "179-wolfram-alpha"
```
Comment (dipamsen) [inline content/pages/challenges/index.json:5]: typo (`179-wolfram-ca`)

**Comment by dipamsen** [comment]
ther is a way to push to pr branches without getting into a fork, but i don't know the canonical way of doing it (using `git`), I just use the VSCode GUI

**Comment by shiffman** [comment]
> ther is a way to push to pr branches without getting into a fork, but i don't know the canonical way of doing it (using `git`), I just use the VSCode GUI

Yes! I was trying to figure this out and got frustrated and gave up. Maybe I should try using the VSCode GUI or GitHub desktop. The message I was getting is that I didn't have permission to push to @kfahn22's fork.

File: content/videos/challenges/179-wolfram-ca/index.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "videoNumber": "179",
   "description": "How is nature hidden in a pile of 0s and 1s? Let's find out by coding a p5.js visualization of the Wolfram Elementary Cellular Automaton!",
   "videoId": "Ggxt06qSAe4",
-  "nebulaSlug": "https://nebula.tv/videos/codingtrain-wolfram-elementary-ca",
+  "nebulaSlug": "git codingtrain-wolfram-elementary-ca",
```
Comment (fturmel) [inline content/videos/challenges/179-wolfram-ca/index.json:6]: bad slug again

**Comment by dipamsen** [comment]
Actually, looks like #1354 doesn't allow maintainers to push changes (it is a setting which is shown while creating a PR).

File: content/pages/challenges/index.json (hunk 1 above)
Comment (dipamsen) [reply content/pages/challenges/index.json:5]: @fturmel is this page not checked by content tests?

**Comment by shiffman** [comment]
Video going live!

File: content/videos/challenges/179-wolfram-ca/index.json (hunk 2 above)
Comment (fturmel) [reply content/videos/challenges/179-wolfram-ca/index.json:6]: @shiffman you missed this issue before merging!

Comment (dipamsen) [reply content/videos/challenges/179-wolfram-ca/index.json:6]: fixed in 6f9e201

**Pull request merged**